src/app/pcasts/dao/series_dao.py

```diff
@@ -1,7 +1,38 @@
+import datetime
 from sqlalchemy.sql.expression import func
 from app.pcasts.dao import episodes_dao
 from . import *
 
+def store_series_and_episodes_from_feed(feed):
+  series_dict = feed.get('series')
+  new_series = Series(
+      id=series_dict.get('id'),
+      title=series_dict.get('title'),
+      country=series_dict.get('country'),
+      author=series_dict.get('author'),
+      image_url_lg=series_dict.get('image_url_lg'),
+      image_url_sm=series_dict.get('image_url_sm'),
+      feed_url=series_dict.get('feed_url'),
+      genres=series_dict.get('genres')
+  )
+  models_to_commit = [new_series]
+  for episode_dict in feed.get('episodes'):
+    ep = Episode(
+        title=episode_dict.get('title'),
+        author=episode_dict.get('author'),
+        summary=episode_dict.get('summary'),
+        pub_date=datetime.datetime.fromtimestamp(episode_dict.get('pub_date')),
+        duration=episode_dict.get('duration'),
+        audio_url=episode_dict.get('audio_url'),
+        tags=episode_dict.get('tags'),
+        series_id=new_series.id)
+    models_to_commit.append(ep)
+  results = db_utils.commit_models(models_to_commit)
+  return results[0] # return the resultant series
+
+def remove_series(series_id):
+  return Series.query.filter(Series.id == series_id).delete()
```

Comment:
Doesn't this need a commit afterwards?

Reply:
No - this is another way of executing a delete that abides by how we setup our ondelete-cascade relationship.  Was gonna mention this to everyone via chat as there might be deletion bugs in our system rn based on how we specified our `ON DELETE CASCADE` referential integrity constraint through `SQLAlchemy`.